server/fakedb/UserParticipatedEvent.csv

```diff
@@ -0,0 +1,22 @@
+﻿userId,eventId
```

Comment:
UserParticipatedEvent -> does this mean this table records past events? If not, we should name this in a better way. Something like EventParticipant? Feel free to explore other options.